.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Add module '...'
+2. Simulate with the configuration '....'
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots/plots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. Macos]
```

Comment:
This list needs to be indented correctly; there is an extra space.

Reply:
Fixed